ctapipe/reco/hillas_reconstructor.py

```diff
@@ -411,19 +420,22 @@ def estimate_core_position(array_pointing, psi, positions):
 
         return core_pos_ground, core_pos_tilted
 
-    @staticmethod
-    def estimate_h_max(cog_vectors, positions):
-        """
-        Estimate the max height by intersecting the lines of the cog directions of each telescope.
+    def estimate_relative_h_max(cog_vectors, positions):
+        """Estimate the relative (to the observatory) vertical height of
+        shower-max by intersecting the lines of the cog directions of each
+        telescope.
 
         Returns
         -------
         astropy.unit.Quantity
-            the estimated max height
+            the estimated height above observatory level (not sea level) of the
+            shower-max point
+
         """
         positions = positions.cartesian.xyz.T.to_value(u.m)
-        # not sure if its better to return the length of the vector of the z component
-        return u.Quantity(
-            np.linalg.norm(line_line_intersection_3d(cog_vectors, positions)),
+        shower_max = u.Quantity(
+            line_line_intersection_3d(cog_vectors, positions),
             u.m,
         )
+
+        return shower_max[2]  # the z-coordinate only
```

Comment:
Could return a dummy `h_max_uncert` here as well.

Reply:
It may need to be computed separately, not sure yet what is the right way to compute it, so so far I'll just leave it.